### PR TITLE
Hello there

### PR DESCRIPTION
**Pull request by Ambroise-lebas**
- Add bootstrap lib
- Update rent_detail.html
- Update global.css

### CODE REVIEW

**Comment by Enzo-Collet** [comment]
Well done Ambroise !

**Review by Enzo-Collet — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -17,7 +27,7 @@ <h2>IESEG Location</h2>
         <a href="#Sign-Up">Sign-Up</a>
       </div>
     </div>
-    <a href="#profile">Profil</a>
+    <a href="#profil">Profil</a>
```
Comment (qblanco) [inline index.html]: why putting profil without an e ? I think the proper way of spelling it with an e

**Review by HeyImMarkus — COMMENTED** [review]
Is it normal that there are over 17000 new lines ? :)

**Review by qblanco — APPROVED** [review]

**Comment by qblanco** [comment]
Markus I think its because he imported a whole library

File: index.html (hunk 1 above)
Comment (Ambroise-lebas) [reply index.html]: Indeed, I'm changing it !

**Pull request merged**